draft-bbs-signatures.md

```diff
@@ -918,11 +924,17 @@ A cryptographic hash function that takes as an arbitrary octet string input and
 hash
 : SHAKE-256 as defined in [@!SHA3].
 
-point\_to\_octets
-: follows the format documented in Appendix C section 1 of [@!I-D.irtf-cfrg-pairing-friendly-curves].
+point\_to\_octets_g1
+: follows the format documented in Appendix C section 1 of [@!I-D.irtf-cfrg-pairing-friendly-curves] for the G1 subgroup.
```

Comment:
```suggestion
: follows the format documented in Appendix C section 1 of [@!I-D.irtf-cfrg-pairing-friendly-curves] for the G1 subgroup, using compression (i.e., setting C\_bit = 1).
```
I think we are using compression in the spec to this point. We can always fall back if need be.